### ISSUE TITLE
cannot find my sample classes

### ISSUE THREAD

**Issue by almohress** [astrelsky/Ghidra-Cpp-Class-Analyzer#45]
I compiled the following code with g++
```cpp
#include <iostream> 
using namespace std;
class Student {
public:
	  int id;
	  string name;
};
int main() {
	Student s1;
	s1.id = 201;
	s1.name = "wellcome";
	cout<<s1.id<<endl;
	cout<<s1.name<<endl;
	return 0;
}
```

Here's g++ version
```bash 
g++ --version
g++ (Ubuntu 10.3.0-1ubuntu1) 10.3.0
```
When I analyzed it with ghidra-cpp-class-analyzer, nothing was detected.
would you help me findout whats going on?

**Comment by astrelsky** [comment]
RTTI is only generated for classed under certain conditions. Your `Student` class does not meet those conditions. In order to force it to be generated you can try using `typeid` on it. Adding a virtual function will also cause RTTI to be created for it.

**Comment by astrelsky** [comment]
Closing due to lack of response and because this is not an issue with the plugin.
